suse_openstack_cloud

```diff
@@ -260,6 +260,12 @@ echo; echo
 find_and_pconf_files /srv/tftpboot/discovery/pxelinux.cfg -type f
 find_and_pconf_files /srv/tftpboot/nodes                  -type f
 
+echo; echo
+find                 /var/lib/crowbar/upgrade -ls
+if ls /va/lib/crowbar/upgrade/*progress.yml >/dev/null 2>&1 ; then
```

Comment:
Typo - has this been tested yet?

Reply:
Oh. (I tested it manually, then obviously broken it)